src/shared/vpick.c

```diff
@@ -85,13 +87,9 @@ static int format_fname(
                         return -ENOMEM;
         }
 
-        if (!strv_isempty(filter->suffix)) {
-                if (strv_length(filter->suffix) > 1)
-                        return -ENOEXEC;
-
+        if (!strv_isempty(filter->suffix))
                 if (!strextend(&fn, filter->suffix[0]))
```

Comment:
Not a matter, but you can merge the two `if` conditions.

Reply:
(I prefer the current form better.)

Reply:
OK. No problem. Feel free to ignore the suggestion.